apis/config/v1beta1/configuration_types.go

```diff
@@ -349,3 +352,28 @@ type ClusterQueueVisibility struct {
 	// Defaults to 10.
 	MaxCount int32 `json:"maxCount,omitempty"`
 }
+
+// +kubebuilder:validation:Enum=LessThanOrEqualToFinalShare;LessThanInitialShare
+type PreemptionStrategy string
+
+const (
+	LessThanOrEqualToFinalShare PreemptionStrategy = "LessThanOrEqualToFinalShare"
+	LessThanInitialShare        PreemptionStrategy = "LessThanInitialShare"
+)
+
+type FairSharing struct {
+	// enabled indicates whether fair sharing is enabled for all cohorts.
+	// Defaults to false.
+	Enabled bool `json:"enabled,omitempty"`
+
+	// preemptionStrategies indicates which constraints should a preemption satisfy.
+	// The preemption algorithm will only use the next strategy in the list if the
+	// incoming workload doesn't fit after using the previous strategies.
```

Comment:
```suggestion
	// The preemption algorithm will only use the next strategy in the list if the
	// incoming workload (preempte) doesn't fit after using the previous strategies.
```

It might be helpful to understand the below description. WDYT?

Reply:
It's preemptor, but fixed.